src/client.ts

```diff
@@ -167,6 +174,9 @@ export class SubscriptionClient {
 
     let opId: string;
 
+    this.activeSubscriptions += 1;
+    this.clearInactivityTimeout();
```

Comment:
If someone is using `lazy = false` it isn't clear to me that this would then get triggered since a request never comes in.